compiler/src/iree/compiler/Dialect/Stream/Conversion/UtilToStream/Patterns.cpp

```diff
@@ -142,6 +142,30 @@ struct ReturnOpConversion
   }
 };
 
+struct UnreachableOpConversion
+    : public OpConversionPattern<IREE::Util::UnreachableOp> {
+  using OpConversionPattern::OpConversionPattern;
+  LogicalResult
+  matchAndRewrite(IREE::Util::UnreachableOp op, OpAdaptor adaptor,
+                  ConversionPatternRewriter &rewriter) const override {
+    rewriter.replaceOpWithNewOp<IREE::Util::UnreachableOp>(op,
+                                                           op.getMessageAttr());
+    return success();
+  }
+};
```

Comment:
What does this pattern do? Does the new op identical to the old op?

Reply:
(same as the other file - thought I'd put more logic in here and never did - good catch :)